dash_app/utils/plotting.py

```diff
@@ -25,6 +25,15 @@ def plot_dataframe(dataframe: pd.DataFrame, max_y_axis: int = None) -> Dict[str,
         for col in dataframe.columns
     ], 'layout': {
         "yaxis": yaxis,
-        "legend": {"orientation": "h"},
-        #"xaxis": {"rangeslider": {}}
+        "xaxis": {"rangeslider": {
+            "visible": True
+        }},
+        "yaxis_title": "Number of people",
+        "title": "Covid 19 projection",
```

Comment:
Maybe we update this layout in the callback itself? Because the plot dataframe should be generic. Or that the plot dataframe has y_title and title as arguments